ssl/statem/statem_srvr.c

```diff
@@ -3694,6 +3699,10 @@ static int create_ticket_prequel(SSL_CONNECTION *s, WPACKET *pkt,
     return 1;
 }
 
+/*
+ * Returns 1 on success, 0 to abort construction of the ticket (non-fatal), or
+ * -1 on fatal error
+ */
```

Comment:
It almost seems that this would map to the `CON_FUNC_RETURN` enum nicely.

Reply:
I've converted this function to use `CON_FUNC_RETURN` too